### PR TITLE
Add Pull Request Club

### PR DESCRIPTION
**Pull request by kyzn**
Hi! I was thinking Pull Request Club could be a nice project to add here. I added it in 2019, but it didn't get a chance back then. Let's try again?

Do let me know if this needs any clarification or updates. Thanks!

### CODE REVIEW

File: perl/PullRequestClub.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+Description
+-----------
+
+[Pull Request Club](https://pullrequest.club/) is a web application where users can sign up with their GitHub accounts. Its purpose is to connect
+ - people who are looking for an open source project to work on, with
+ - people who want new contributors for their open source projects.
+
+Pull Request Club gives out monthly assignments. Web application is built on Catalyst (Perl), and it has a SQLite database.
+
```
Comment (JJ) [inline perl/PullRequestClub.md:12]: You should maybe move a few "expected outcomes" here as objectives. A project, to be picked up by a student, should have a clear objective or sets of objectives. If you've got a roadmap for the site, point at it; if there are a few milestones, that would help too.

File: perl/PullRequestClub.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+Expected outcomes
+-----------------
+
+Improve landing page to include some statistics such as "how many active assignments are there" and "how many assignments were completed". Build another page that displays monthly stats for curious users/researchers.
+
+Required skills
+---------------
+
+Experience with Perl & SQL. Knowledge about web applications.
```
Comment (JJ) [inline perl/PullRequestClub.md]: Please specify a bit more here. Client-server? Web services?  MVC?

**Review by JJ — CHANGES_REQUESTED** [review]
Please check out my comments, and we're good to go.

**Comment by JJ** [comment]
Do you want to give this a one-over before we close?

**Comment by kyzn** [comment]
Hey JJ! Thanks for the reminder, and apologies for the wait. Let me spend some time on this on weekend, and update the branch.

**Comment by kyzn** [comment]
Hey @JJ, thanks for waiting. I've updated the details, can you let me know if it looks good? Thanks!

**Review by JJ — APPROVED** [review]
Looks fine. Thanks!

**Pull request merged**